recipes/nfir/all/conanfile.py

```diff
@@ -0,0 +1,56 @@
+from conan import ConanFile
+from conan.tools.cmake import cmake_layout, CMake, CMakeToolchain
+from conan.tools.files import copy, get, collect_libs
+from conan.tools.files import apply_conandata_patches
+from os import path
+
+class NFIRConan(ConanFile):
+    name = "nfir"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/usnistgov/NFIR"
+    topics = ("NIST, NFIR")
+    license = "NIST"
+    description = "The NIST Fingerprint Image Resampler NFIR is a library capable of upsampling or downsampling fingerprint images."
+    settings = "os", "compiler", "build_type", "arch"
+    options = { "shared": [True, False] }
+    default_options = { "shared": False }
+    generators = "CMakeDeps"
+    package_type = "library"
+    exports_sources = "CMakeLists.txt", "patches/*"
```

Comment:
```suggestion
    implements = ["auto_shared_fpic"]
```
`exports_sources` is unnecessary on CCI. `auto_shared_fpic` is necessary for the correct handling of `shared` and `fPIC` options: https://docs.conan.io/2/reference/conanfile/methods/configure.html#auto-shared-fpic